lib/rbvmomi/basic_types.rb

```diff
@@ -201,6 +199,36 @@ def _ref
         @ref
       end
 
+      def to_s
+        "#{self.class.wsdl_name}(#{@ref.inspect})"
+      end
+
+      def to_hash
+        to_s
+      end
+
+      def pretty_print pp
+        pp.text to_s
+      end
```

Comment:
Wonder if this should move to base, since it doesn't care about ref nor connection?  (not for this PR)

Reply:
If we override `inspect` we could drop the `pretty_print` here altogether.  What we don't want is to dump the entire connection object which is enormous (includes the whole vmodl.db hash).